hazelcast/src/main/java/com/hazelcast/quorum/QuorumChangeStateOperation.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.quorum;
+
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.spi.Operation;
+import java.io.IOException;
+
+/**
+ * Changes quorum state to the given state on the executing node.
+ */
+public class QuorumChangeStateOperation extends Operation {
```

Comment:
this package breaks from our general package layout. 

I would expect:
```
com.hazelcast.quorum for api
com.hazelcast.quorum.impl for most of the implementing stuff
com.hazelcast.quorum.impl.operations for the operations
```

The most important part is that the com.hazelcast.quorum package does not contains any code that should not be visible to the end user.